core/src/backend/ui.rs

```diff
@@ -145,6 +158,10 @@ impl UiBackend for NullUiBackend {
 
     fn set_mouse_cursor(&mut self, _cursor: MouseCursor) {}
 
+    fn clipboard_content(&mut self) -> String {
+        "".to_string()
```

Comment:
(Nit: Maybe insert a `TODO` or `FIXME` comment here?).

Reply:
No, this is the NullUiBackend, which doesn't have a clipboard